jee-commons/src/main/java/biz/paluch/jee/commons/BeanLookup.java

```diff
@@ -10,9 +10,14 @@
 public final class BeanLookup {
 
     /**
-     * Default JNDI name of the BeanManager.
+     * Default JNDI names of the BeanManager.
      */
-    public static final String BEANMANAGER_JNDI_NAME = "java:comp/BeanManager";
+    public static final String[] BEANMANAGER_JNDI_NAMES = {"java:comp/BeanManager", "java:comp/env/BeanManager"};
```

Comment:
As this is just some convention over configuration behaviour to detect bean managers of common application servers, I don't think that it's an important feature. If one has such exotic JNDI names, maybe it's better to implement an own bean manager detection and use setDefaultBeanManager().

But yes, why not.